packages/frontend/src/components/Dialog/DialogHeader.tsx

```diff
@@ -9,24 +9,54 @@ import EditButton from './EditButton'
 import type { DialogProps } from '../../contexts/DialogContext'
 
 import styles from './styles.module.scss'
+import Button from '../Button'
+import Icon from '../Icon'
+import useTranslationFunction from '../../hooks/useTranslationFunction'
 
 type Props = PropsWithChildren<{
   title?: string
   onClickBack?: () => void
   onClickEdit?: () => void
   onClose?: DialogProps['onClose']
+  onContextMenuClick?: (_event: any) => void
   dataTestid?: string
 }>
 
 export default function DialogHeader(props: Props) {
-  const { onClickBack, title, onClose, onClickEdit, children } = props
+  const {
+    onClickBack,
+    title,
+    onClose,
+    onClickEdit,
+    onContextMenuClick,
+    children,
+  } = props
   const dataTestid = props.dataTestid ?? 'dialog-header'
+  const tx = useTranslationFunction()
   return (
     <header className={classNames(styles.dialogHeader)}>
       {onClickBack && (
         <BackButton onClick={onClickBack} data-testid={dataTestid + '-back'} />
       )}
       {title && <DialogHeading>{title}</DialogHeading>}
+      {onContextMenuClick && (
+        <span
+          style={{
+            marginLeft: 0,
+            marginRight: '3px',
+          }}
+          data-no-drag-region
+        >
+          <Button
+            className={classNames(styles.headerThreeDotButton)}
+            aria-label={tx('menu_more_options')}
+            data-testid={dataTestid + '-context-menu'}
+            onClick={onContextMenuClick}
+          >
+            <Icon coloring='contextMenu' icon='more' rotation={90} size={24} />
+          </Button>
+        </span>
+      )}
```

Comment:
One cannot set up a proxy when importing a backup and adding a second device though.

Reply:
Since the settings are also imported from backup it would not make sense to set it before importing a backup.

Adding as second device might work if you add the proxy settings and then go back to the initial screen top open the Add as second device dialog.

The behaviour is just copied from Android atm

Reply:
> Since the settings are also imported from backup it would not make sense to set it before importing a backup.

Hmm, I didn't know that, interesting.

OK, let's mark as resolved.